Modules/Loadable/Markups/MRML/vtkMRMLMarkupsPlaneNode.h

```diff
@@ -39,9 +39,9 @@
 /// Markups is intended to be used for manual marking/editing of point positions.
 ///
 /// Coordinate systems used:
-///   - Local: Local coordinates
+///   - Node: Local node coordinates
 ///   - World: All parent transforms on node applied to local.
-///   - Plane: Plane coordinate space (Origin of plane at 0,0,0, XYZ axis aligned to XYZ unit vectors).
+///   - Object: Plane coordinate space (Origin of plane at 0,0,0, XYZ axis aligned to XYZ unit vectors).
```

Comment:
Origin by definition is (0,0,0), so that's not helping. Is the origin the center of the rectangle?

Specify that X and Y axes are parallel to the first and second axes of the plane's rectangle and Z axis is the plane normal.